features/enable_fips_vm.feature

```diff
@@ -8,7 +8,12 @@ Feature: FIPS enablement in lxd VMs
     Scenario Outline: Attached enable of FIPS in an ubuntu lxd vm
         Given a `<release>` machine with ubuntu-advantage-tools installed
         When I attach `contract_token` with sudo
-        And I run `ua disable livepatch` with sudo
+        When I run `ua status --format json` with sudo
+        Then stdout matches regexp:
+        """
+        \"blocked_by\": \[{\"name\": \"livepatch\", \"reason_code\": \"livepatch\-invalidates\-fips\", \"reason\": \"Livepatch cannot be enabled while running the official FIPS certified kernel. If you would like a FIPS compliant kernel with additional bug fixes and security updates, you can use the FIPS Updates service with Livepatch.\"}\]
```

Comment:
This will match the `"blocked_by"` for the `fips` service. So it is saying that `fips` is blocked/cannot be enabled because "livepatch` is enabled.

Reply:
Though it is hard to tell that here, because I didn't include the name of the service that the "blocked_by" belongs to. I should change the integration tests to include that so it is clearer